### PR TITLE
JDBC-53 Improve test suite reporting of converted vals

### PR DESCRIPTION
**Pull request by rpastrana**
Signed-off-by: Pastrana, Rodrigo <rodrigo.pastrana@lexisnexis.com>

### CODE REVIEW

**Comment by rpastrana** [comment]
@Michael-Gardner please review.

File: src/main/java/org/hpccsystems/jdbcdriver/HPCCJDBCUtils.java [hunk 1]
```diff
@@ -797,8 +797,15 @@ else if (java.math.BigDecimal.class == clazz)
             return new BigDecimal(value.replaceAll(",", ""));
         else if (java.sql.Date.class == clazz) //ASSUMES yyyy-MM-dd!!!
             return java.sql.Date.valueOf(value);
-        else if (java.sql.Time.class == clazz)//ASSUMES HH:mm 12 hour format!!!
-            return new java.sql.Time(((java.util.Date)HoursMinutesSeconds12HourFormat.parse(value)).getTime());
+        else if (java.sql.Time.class == clazz)//ASSUMES HH:mm:ss!!!
+        try
+        {
+            return new java.sql.Time(((java.util.Date)HoursColonMinutesColonSeconds24HourFormat.parse(value)).getTime());
```
Comment (Michael-Gardner) [inline src/main/java/org/hpccsystems/jdbcdriver/HPCCJDBCUtils.java]: What happens here if the exception is thrown in the constructor or getTime() method?  What object type gets returned and is it guaranteed behavior.

**Comment by Michael-Gardner** [comment]
Overall everything looks good.  Just the one comment.

**Comment by rpastrana** [comment]
@Michael-Gardner please review.

**Pull request merged**